### ISSUE TITLE
Элементарное поле ДА/НЕТ

### ISSUE THREAD

**Issue by iAndroS** [instantsoft/icms2#671]
Наверное, много кто сталкивался с необходимостью выводить информацию в виде названия поля и рядом надписи да или нет. Приходится мудрить выпадающий список с пустым значением. Нужно же всего лишь сделать поле с ползунком в обе стороны (по аналогии с мобильным переключателем-слайдером), которое при создании новой записи было бы неактивированным, но обязательным к заполнению. Этот момент здорово помог бы в выводе нужной информации в каталогах, а также помог бы избежать случайно незаполненного поля при создании новой записи. Казалось бы, простое решение, использовать для полей такого типа единственное поле - список-мультивыбор с выбором нужных значений, но в таком случае пользователь в записи не видит важной информации, в случае если чекбокс в мультисписке был не отмечен (значение НЕТ). Согласитесь, было бы важным выводить информацию в виде "Сертификация госорганами - НЕТ"?

**Comment by iAndroS** [comment]
Кстати, подумалось мне тут - проблему можно было бы решить без создания нового поля, но правкой шаблона вывода неотмеченных значений в виде полупрозрачных надписей или отмеченные  помечать иконкой-галочкой какой-то

**Comment by fuzegit** [comment]
Есть поле "Флаг", которое и выводит да/нет

**Comment by iAndroS** [comment]
Да, выводит, но только в фильтре. В записи же не видно, что какая-то функция/параметр записи имеет значение НЕТ, в некоторых случаях это критично важно.
